soroban-settings/phase1.json

```diff
@@ -295,8 +295,8 @@
     {
       "state_archival": {
           "max_entry_ttl": 3110400,
-          "min_temporary_ttl": 16,
-          "min_persistent_ttl": 120960,
+          "min_temporary_ttl": 17000,
```

Comment:
Do we want to do `17280` instead so we have exactly 24 hours?

Reply:
This number may have been chosen as a result of some calculation, so I'll defer to @dmkozh.

Reply:
17280 sounds fine to me - I just used a few rough numbers to compare the options.

Reply:
Update to 17280